tests/test_custom_models.py

```diff
@@ -350,6 +351,18 @@
         VeraConfig,
         {"target_modules": ["conv1d"]},
     ),
+    ########
+    # FourierFT #
+    ########
+    ("Vanilla MLP 1 FourierFT", "MLP", FourierConfig, {"n_frequency": 10, "target_modules": "lin0"}),
+    ("Vanilla MLP 2 FourierFT", "MLP", FourierConfig, {"n_frequency": 10, "target_modules": ["lin0"]}),
+    ("Vanilla MLP 3 FourierFT", "MLP", FourierConfig, {"n_frequency": 10, "target_modules": ["lin1"]}),
+    (
+        "Vanilla MLP 5 FourierFT",
+        "MLP",
+        FourierConfig,
+        {"n_frequency": 10, "target_modules": ["lin0"], "modules_to_save": ["lin1"]},
+    ),
```

Comment:
Can we add an example where we target set `"target_modules": ["lin0", "lin1"]`? Also, another similar example where we additionally set different frequencies for the two layers via `n_frequency_pattern`.